features/step_definitions/paypal_steps.rb

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+
+def dismiss_onboarding_wizard_dialog
+  expect(page).to have_content("Woohoo, task completed!")
+  page.click_on("I'll do it later, thanks")
+end
```

Comment:
Should this be in section module? Section::Wizard, Section::OnboardingPopup, or something like that?